incubator/hnc/docs/releasing.md

```diff
@@ -112,9 +112,19 @@ this](https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-se
 ## Test
 
 Test! At a minimum, install it onto a cluster, download the kubectl plugin, and
-ensure that you can create a subnamespace. Better yet, run through the demo and
-all the `hack/test-*.sh` scripts. _TODO: automate this
-[#785](https://github.com/kubernetes-sigs/multi-tenancy/issues/785)_
+run:
+
+```bash
+export HNC_REPAIR=<path to installable YAML; https is fine>
+go clean -testcache
+go test -v -timeout 0 ./test/...
```

Comment:
Why not `make e2e-test` after `export`?

Reply:
Because I was being lazy :stuck_out_tongue: I've update the makefile and changed the instructions.